frontends/pytorch/README.md

```diff
@@ -13,5 +13,40 @@ along with their lowerings to common intermediate dialects and backends. This
 directory should be purely about interfacing with the PyTorch/LibTorch
 components for extracting and executing programs.
 
-See the [overall documentation for frontends](../README.md) for further details
-about code layout and integration philosophy.
+The code in this directory is intended to integrate tightly with pytorch, and
+follows the code style for pytorch.  See the [overall documentation for
+frontends](../README.md) for further details about code layout and integration
+philosophy.
+
```

Comment:
Can you add a note: "This directory exists to provide a working frontend to an MLIR based pytorch compilation flow and is not intended to be contributed to the LLVM monorepo. If the project is successful, it makes more sense to either break it out as an independent project that depends on LLVM/MLIR/npcomp or contribute it upstream to PyTorch. However, as it will be quite some time before the components are in a state to support such a dependency, it is being carried in-tree in the interim."